### ISSUE TITLE
[Docs] Deprecated symbols are switched in 2.18.0 release notes

### ISSUE THREAD

**Issue by TacoTheDank** [google/ExoPlayer#10363]
As seen here: https://github.com/google/ExoPlayer/releases#:~:text=Remove%20Player.Listener.onTracksChanged.%20Use%20Player.Listener.onTracksInfoChanged%20instead

It currently says ``Remove Player.Listener.onTracksChanged. Use Player.Listener.onTracksInfoChanged instead.``

It SHOULD say ``Remove Player.Listener.onTracksInfoChanged. Use Player.Listener.onTracksChanged instead.``

I believe that [this line](https://github.com/google/ExoPlayer/releases#:~:text=Remove%20Player.getCurrentTrackGroups%20and%20Player.getCurrentTrackSelections.%20Use%20Player.getCurrentTracksInfo%20instead.%20You%20can%20also%20continue%20to%20use%20ExoPlayer.getCurrentTrackGroups%20and%20ExoPlayer.getCurrentTrackSelections%2C%20although%20these%20methods%20remain%20deprecated.) is also wrong: As evident in the library, `Player.getCurrentTracksInfo` was renamed. It should not say "use this instead." I believe there are also other errors in this bullet, though I cannot say for sure.

These are just the ones I caught; however, I would suggest to have the entire 2.18.0 changelog reread in case there are also other mistakes that need fixing.

Thank you.

**Comment by icbaker** [comment]
Hmmm, interesting.

Both those release notes were added in https://github.com/google/ExoPlayer/commit/25e58f5f2a6c15138daef4103ab2aff711e662ff (Feb 2022) and it matches the content of the commit (i.e. `Player.Listener.onTracksChanged` is removed, `onTracksInfoChanged` is preserved).

However we then made a follow-up commit (https://github.com/google/ExoPlayer/commit/0ad508b14f232a2d9551cdead61c3e7fa9c0e9e8) in April that renamed `TracksInfo` to `Tracks` and therefore, as collateral, renamed `onTracksInfoChanged` (back) to `onTracksChanged`. This is documented in separate release note bullet. 

Both these commits were made after the 2.17.0 release, so they're both included in 2.18.0. Unfortunately we didn't merge the two release note bullets together, so the result is (as you've pointed out) slightly nonsensical.

Thanks for pointing this out! I think this is specific to this case where we removed and later re-added some symbols, rather than a systematic problem with the 2.18.0 release notes but I'll take a look for other issues too and make a change to fix things to make a bit more sense.
